core/trace/src/main/scala-2/org/typelevel/otel4s/trace/TracerMacro.scala

```diff
@@ -101,10 +102,11 @@ private[otel4s] trait TracerMacro[F[_]] {
     * @param attributes
     *   the set of attributes to associate with the span
     */
-  def resourceSpan[A](name: String, attributes: Attribute[_]*)(
-      resource: Resource[F, A]
-  ): SpanOps.Aux[F, Span.Res[F, A]] =
-    macro TracerMacro.resourceSpan[F, A]
+  def spanResource(
```

Comment:
Note for later: "The structure of the inner spans" in the scaladoc is no longer correct, right?

Reply:
correct. but the method is going to be deleted anyway, as it doesn't actually serve a purpose. it's identical to `span(name, attributes: _*).resource`, which is only one character longer. so I cba